lego/apps/users/models.py

```diff
@@ -194,20 +194,24 @@ class Meta:
     @abakus_cached_property
     def memberships(self):
         return Membership.objects.filter(
-            deleted=False,
```

Comment:
Is it intended to remove this deleted kwarg, but still have it in`all_groups` below?

Reply:
Ahh, guess it can be removed from `all_groups` as well. `deleted=False` is implicit because the default model manager is `PersistentModelManager`, that only returns the items with `deleted=False`